modules/fluid_properties/unit/src/TabulatedBicubicFluidPropertiesTest.C

```diff
@@ -204,6 +204,15 @@ TEST_F(TabulatedBicubicFluidPropertiesTest, fromFileVE)
     REL_TEST(k1, k2, 0.001);
   }
 
+  // check computation of fluid props from p, s
+  {
+    Real s = _tab_fp_ve->s_from_p_T(p, T);
+
+    Real rho1 = _tab_fp_ve->rho_from_p_T(p, T);
+    Real rho2 = _tab_fp_ve->rho_from_p_s(p, s);
+    REL_TEST(rho1, rho2, 0.001);
```

Comment:
Any test for T_from_p_s and its derivatives?

Reply:
Added some. I guess it s future proofing if someone ever manages to disconnect the two